app/swagger/swagger/requests/claim_letters.rb

```diff
@@ -69,6 +69,7 @@ class ClaimLetters < ApplicationController
         end
         property :version, type: :string
         property :type_description, type: :string
+        property :display_description, type: :string
```

Comment:
I'm flying blind on this one, I'm not sure if this file is automated or if I need to change it or what.

Reply:
I would go ahead and call this `display_name` as that is used elsewhere in CST

Reply:
After [this conversation](https://dsva.slack.com/archives/C018V2JCWRJ/p1730752891087899?thread_ts=1730749381.713609&cid=C018V2JCWRJ) I am going to reappropriate the `type_description`, actually. That's what mobile is displaying, so I will simply change that field to be the human-readable name.

Does that sound ok to you?